### ISSUE TITLE
Names above Shulkers dont turn off via config

### ISSUE THREAD

**Issue by spacialspud** [JackFred2/WhereIsIt#40]
**Mod Version**
2.5.0+1.21

**Describe the bug**
The display tag above named containers wont turn off when disabled in the settings

**Expected behavior**
The display tag to turn off when disabled

**To Reproduce**
rename shulker box and disable over block name display (It doesnt disable it)

**Config (Recommended)**
`
{
	client: {
		search_using_item_in_hand: false,
		play_sound_on_request: true,
		close_gui_on_found_results: true,
		show_slot_highlights: true,
		slot_highlight_mouse_factor: 1.0,
		slot_highlight_x_factor: 1.0,
		highlight_time_factor: 1.0,
		show_container_names_in_results: false,
		container_name_label_scale: 1.00,
		random_scheme: false,
		colour_scheme: "SOLID",
		solid_colour: -16720385,
		compatibility: {
			recipe_book_support: true,
			jei_support: true,
			rei_support: true,
			emi_support: true
		},
		debug: {
			print_search_requests_in_chat: false
		}
	},
	common: {
		search_range_blocks: 8,
		do_nested_search: true,
		command_aliases: [
			"whereis"
		],
		fadeout_time_ticks: 300,
		debug: {
			enable_default_searchers: true,
			print_search_time: false,
			force_server_side_highlights_only: false
		}
	},
	server: {
		rate_limit: true
	}
}
`

**Comment by JackFred2** [comment]
Do you have Chest Tracker installed? The names above chests / shulkers that are always there (outside of searching) are part of that mod's memory bank settings.On 1 Aug 2024 06:34, spacialspud ***@***.***> wrote:
Mod Version
2.5.0+1.21
Describe the bug
The display tag above named containers wont turn off when disabled in the settings
Expected behavior
The display tag to turn off when disabled
To Reproduce
rename shulker box and disable over block name display (It doesnt disable it)
Config (Recommended)
{ client: { search_using_item_in_hand: false, play_sound_on_request: true, close_gui_on_found_results: true, show_slot_highlights: true, slot_highlight_mouse_factor: 1.0, slot_highlight_x_factor: 1.0, highlight_time_factor: 1.0, show_container_names_in_results: false, container_name_label_scale: 1.00, random_scheme: false, colour_scheme: "SOLID", solid_colour: -16720385, compatibility: { recipe_book_support: true, jei_support: true, rei_support: true, emi_support: true }, debug: { print_search_requests_in_chat: false } }, common: { search_range_blocks: 8, do_nested_search: true, command_aliases: [ "whereis" ], fadeout_time_ticks: 300, debug: { enable_default_searchers: true, print_search_time: false, force_server_side_highlights_only: false } }, server: { rate_limit: true } }

**Comment by spacialspud** [comment]
Yes I had chest tracker as I was using that to search for items. I went into both configs and turnt off the names for chests but it didnt turn off for Whereisit no matter what I did.
